Lib/test/test_dis.py

```diff
@@ -1470,195 +1470,209 @@ def _prepare_test_cases():
-  Instruction(opname='JUMP_BACKWARD', opcode=140, arg=52, argval=276, argrepr='to 276', offset=378, starts_line=None, is_jump_target=False, positions=None),
-  Instruction(opname='RERAISE', opcode=119, arg=0, argval=0, argrepr='', offset=380, starts_line=22, is_jump_target=True, positions=None),
-  Instruction(opname='COPY', opcode=120, arg=3, argval=3, argrepr='', offset=382, starts_line=None, is_jump_target=False, positions=None),
-  Instruction(opname='POP_EXCEPT', opcode=89, arg=None, argval=None, argrepr='', offset=384, starts_line=None, is_jump_target=False, positions=None),
-  Instruction(opname='RERAISE', opcode=119, arg=1, argval=1, argrepr='', offset=386, starts_line=None, is_jump_target=False, positions=None),
-  Instruction(opname='PUSH_EXC_INFO', opcode=35, arg=None, argval=None, argrepr='', offset=388, starts_line=None, is_jump_target=False, positions=None),
-  Instruction(opname='LOAD_GLOBAL', opcode=116, arg=3, argval='print', argrepr='NULL + print', offset=390, starts_line=28, is_jump_target=False, positions=None),
-  Instruction(opname='LOAD_CONST', opcode=100, arg=10, argval="OK, now we're done", argrepr='"OK, now we\'re done"', offset=402, starts_line=None, is_jump_target=False, positions=None),
-  Instruction(opname='CALL', opcode=171, arg=1, argval=1, argrepr='', offset=404, starts_line=None, is_jump_target=False, positions=None),
-  Instruction(opname='POP_TOP', opcode=1, arg=None, argval=None, argrepr='', offset=414, starts_line=None, is_jump_target=False, positions=None),
-  Instruction(opname='RERAISE', opcode=119, arg=0, argval=0, argrepr='', offset=416, starts_line=None, is_jump_target=False, positions=None),
-  Instruction(opname='COPY', opcode=120, arg=3, argval=3, argrepr='', offset=418, starts_line=None, is_jump_target=False, positions=None),
-  Instruction(opname='POP_EXCEPT', opcode=89, arg=None, argval=None, argrepr='', offset=420, starts_line=None, is_jump_target=False, positions=None),
-  Instruction(opname='RERAISE', opcode=119, arg=1, argval=1, argrepr='', offset=422, starts_line=None, is_jump_target=False, positions=None),
-]
+def inst(opname, opcode, arg, argval, argrepr, offset, starts_line, is_jump_target):
+    return dis.Instruction(opname=opname, opcode=opcode, arg=arg, argval=argval,
+                           argrepr=argrepr, offset=offset,
+                           starts_line=starts_line, is_jump_target=is_jump_target,
+                           positions=None)
+
+_ = None
+
+expected_opinfo_outer = [inst(*args) for args in [
+    # opname, opcode, arg, argval, argrepr, offset, starts_line, is_jump_target
+    ('MAKE_CELL'                 , 135,   0,                  'a',                       'a',    0,    _,  False),
+    ('MAKE_CELL'                 , 135,   1,                  'b',                       'b',    2,    _,  False),
+    ('RESUME'                    , 151,   0,                    0,                        '',    4,    1,  False),
```

Comment:
It might be even more readable as "opname (opcode), org, argval (argrepr), offset, starts_line, is_jump_target".

Something like:
```
('MAKE_CELL (135)',   1,                  'b' ('b'),    2,    _,  False),
('RESUME (151)',      0,                   0 (''),      4,    1,  False),
```

Reply:
i gave that a try, but it complicated the code to convert the table into `Instruction`s, which i figured was not worth it.